config/xlcore.sh

```diff
@@ -1,2 +1 @@
-#This line will need to be changed each time the XLCore devs update bundled wine.
-XLCORE_WINE_VERSION="wine-xiv-staging-fsync-git-7.10.r3.g560db77d"
+XLCORE_WINE_VERSION=$(ls -1tr | tail -n1)
```

Comment:
This still isn't correct, since this will eval immediately on script sourcing and not when the var is used. Probably the entire file can be dropped and this can be put inline with a comment or something, or this var could be set to empty by default to allow overriding, and where it's used check for empty and if empty run the ls to figure out latest version.